pkg/models/settings.go

```diff
@@ -9,6 +9,7 @@ import (
 // Settings represents the Datasource options in Grafana
 type Settings struct {
 	AccessToken    string `json:"accessToken"`
+	GithubUrl string `json:"githubUrl"`
```

Comment:
mind running `gofmt` in the project for me?

Should clear up formatting issues like this one.

Reply:
Formatted now. New to golang, thanks for the tip!